api/outdated/models.py

```diff
@@ -57,11 +60,29 @@ def pre_save(self, model_instance, add):
         return super().pre_save(model_instance, add)
 
 
+def validate_repo_exists(value: str) -> None:
+    """Validate the existance of a remote git repository."""
+    url = "https://" + value
+
+    if value.startswith("file://") and settings.ENV == "test":
+        url = value
```

Comment:
What's the reasoning behind not storing the scheme in the DB?

Reply:
Repos without ssl are out of scope, and `file://` will only ever be used in tests

See:
- https://github.com/adfinis/Outdated/issues/91 
- https://github.com/adfinis/Outdated/issues/93
- https://github.com/adfinis/Outdated/pull/92#issuecomment-1715624976

Reply:
I see. Fine by me :+1: Just as an input, if you plan to extend this in the future: If you store the scheme in a different model field, validation for uniqueness won't be more difficult than now.